packages/desmos-std/src/msg.rs

```diff
@@ -1,74 +1,57 @@
-use crate::types::DesmosRoute;
 use cosmwasm_std::{CosmosMsg, CustomMsg};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
+use crate::subspaces::msg::SubspacesMsg;
```

Comment:
better to refactor imports here and put the common ones together when possible.
Like I did it here:
https://github.com/desmos-labs/desmos-contracts/blob/05abb9fb4a38661f2586dded254465e25bfa7c7f/packages/desmos-std/src/profiles/mock.rs#L2